vita3k/renderer/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ add_library(
 )
 
 target_include_directories(renderer PUBLIC include)
-target_link_libraries(renderer PUBLIC crypto display dlmalloc mem stb shader glutil threads config util vkutil)
+target_link_libraries(renderer PUBLIC crypto display dlmalloc mem modules stb shader glutil threads config util vkutil)
```

Comment:
Sorry, but we really shouldn't have to make renderer dependant on modules. Maybe put the tracy files somewhere else like util ?

Reply:
done